src/hooks/useNotification.ts

```diff
@@ -0,0 +1,24 @@
+import { NotificationType } from '@/libs/constants'
+import { useNotificationStore } from '@/stores/notification'
+
+const useAddNotification = () => {
```

Comment:
Please use a general noun for naming a hook, for this case, `useNotification `will an okay name.
If we use `useAddNotification `hook, it look like 
`const {addNotification} = useAddNotification()`
look kind of dummy.
Moreover, in some cases, we will need some utility such as `currentNotification`, `isNotification`, `getHistoryNotification`. It's more pro when we hold all the utility function inside that hook. Then, it's gonna be 
`const { addNotification, currentNotification, isNotification, getHistoryNotification } = useNotification()`
It's look more pro 😎